### ISSUE TITLE
Documentation of exported GDScript variables does not appear in Inspector

### ISSUE THREAD

**Issue by glevner** [godotengine/godot#96084]
### Tested versions

- Reproducible in 4.3.stable.

### System information

Godot v4.3.stable - macOS 14.6.1 - GLES3 (Compatibility) - Apple M3 - Apple M3 (8 Threads)

### Issue description

When opening an existing project, comments beginning with "##" for exported variables in GDScript are not displayed in the Inspector panel, which just says _No description available_. If you resave the relevant script file, the documentation reappears.

### Steps to reproduce

Create a new project. Create a Node root node, and add a script to the node:

```
extends Node

## Oh, look! An exported variable!
@export var glorp = 0
```

The variable appears in the Inspector panel, but the property's tooltip initially says _No description available_. If you save the script file, the documentation appears in the Inspector panel.

Now, **Quit to Project List** and reload the project. The property's tooltip again says _No description available_. You must make a change to the script file and resave it for the documentation to reappear.

### Minimal reproduction project (MRP)

N/A

**Comment by Hilderin** [comment]
I think this issue will be fixed by #95821

**Comment by dalexeev** [comment]
Duplicate of:

* #72406
* #86577

Thanks for reporting the bug nonetheless!
